CHANGELOG.rst

```diff
@@ -5,6 +5,15 @@ CHANGELOG
 Unreleased
 ==========
 
+2.13.0
```

Comment:
a minor version bump is typically reserved for rolling out enhancements or new features. The below changes are either bugfixes or minor clean up, so IMO this should be a patch version release.